cypress/integration/lesson15.spec.js

```diff
@@ -0,0 +1,493 @@
+import { faker } from '@faker-js/faker';
+import { getInnerText, getNumbers } from '../../utilities/data';
+import {
+  sortNumbersASC,
+  sortStringsASC,
+  sortNumbersDESC,
+  sortStringsDESC,
+} from '../../utilities/sorting';
+
+const firstName = faker.name.firstName();
+const lastName = faker.name.lastName();
+const email = faker.internet.email();
+const age = faker.random.number({ min: 0, max: 100, precision: 1 });
+const salaryDollars = faker.random.number({
+  min: 400,
+  max: 100000,
+  precision: 1,
+});
+const department = faker.lorem.words(1);
+const randomNumber = faker.random.number({ min: 1, max: 100, precision: 1 });
+
+describe('Lesson15', () => {
+  const firstUserCellsSelector =
+    '.rt-table .rt-tr-group:first-child .rt-tr .rt-td';
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/webtables');
+  });
+
+  describe('User creating', () => {
+    it('Should open registration form', () => {
```

Comment:
U can use following structure:

```suggestion
describe('Lesson15', () => {
  const firstUserCellsSelector =
    '.rt-table .rt-tr-group:first-child .rt-tr .rt-td';
  beforeEach(() => {
    cy.visit('https://demoqa.com/webtables');
  });

  context('User creating', () => {
    it('Should open registration form', () => {
```

Reply:
Have changed, thanks